client/src/components/ItemList/itemlist.style.js

```diff
@@ -9,15 +9,15 @@ export const List = styled.div`
 `;
 
 export const StyledHeader = styled.h1`
-font-size: 20px;
-font-family:'Assistant';
-background-color:#FAFAFA;
-padding: 22px 0px;
-text-align:left;
-Margin-left:20px;
-@media (max-width: 650px) {
-  font-size: 16px;
-}
+  font-size: 20px;
+  font-family: 'Assistant';
```

Comment:
We don't need this? As it's on the body styles.

Reply:
okie